src/db/vco_mission_text.loc.tsv

```diff
@@ -393,10 +394,12 @@ mission_text_text_vco_ogr_gre_1_rich_walk_99	The Goldtooth tribe is notorious fo
 mission_text_text_vco_ogr_gre_1_rich_walk	The Goldtooth tribe is notorious for its wealth. Ever eager to show off their top status, they are given to ostentatious displays - from a vast mountaintop feast hall, to gem-encrusted gutplates and even a solid gold Mawtooth. Now, under the rule of Greasus, the tribe will grow richer still.\\nHave a gross income of 25,000.	true
 mission_text_text_vco_ogr_gre_2_overtyrants_reign_1	Greasus' overflowing coffers have not diminished his greed nor his all-consuming desire to conquer everything he sees. The Tribestealer will unite the Ogres of the Mountains of Mourn into the largest Ogre Kingdom in history!	true
 mission_text_text_vco_ogr_gre_2_overtyrants_reign_2	<dummy>The Black Orc known as Grimgor Ironhide seeks to challenge Greasus' reign, brazenly dubbing himself "Boss of the East". The Sceptre of Titans will disabuse him of this notion.	true
+mission_text_text_vco_ogr_gre_2_at_least_6_provinces	Control at least 6 of the following provinces either by direct ownership or through [[img:icon_vassal]][[/img]]vassals and [[img:icon_military_alliance]][[/img]]military allies:	true
```

Comment:
No change required because I already recorded these ones. Just a reminder to to follow the alphabetical order, as you had one grimgor key in the middle of the Greasus ones.